packages/st2/debian/st2actionrunner.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=StackStorm service st2actionrunner
+After=network.target
+
+[Service]
+Type=oneshot
+EnvironmentFile=-/etc/sysconfig/st2actionrunner
```

Comment:
This is used by `RedHat`-like systems,
for `Debian`-like systems the directory with default environment variables is `/etc/default`

BTW, see: https://github.com/StackStorm/st2chatops/issues/50